packages/common/src/components/utils/Modal/PopUpModal.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { ModalBase } from '../..'
+
+interface Props {
+  open: boolean
+  modalTitle: string
+}
+
+const PopUpModal: React.FC<Props> = ({ open, modalTitle, children }) => {
+  if (!open) {
+    return null
+  }
+
+  const handleClose = () => {
+    return null
+  }
```

Comment:
Why return `null`? If it's a no-op the perhaps this would be clearer
```suggestion
  const handleClose = () => {}
```